client/src/components/filmItem.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+export default function FilmItem({ film }) {
+  return (
+    <div className='filmItem'>
+      <div className='filmHead'>
+        <h2>{film.title}</h2>
+        <h3>{film.release_date}</h3>
+      </div>
+      <div className='openingCrawl'>
+        <span>Opening Crawl</span>
```

Comment:
I know the assignment talks in terms of **crawls**, but most people would understand this to be a **synopsis**. But I would probably leave out this `span` altogether because I think it would be immediately clear to users what this text represents. But that's just me, I'm of the less-is-more school.

Reply:
you are right, I removed that span completely.